### PR TITLE
Automated code format update

### PR DESCRIPTION
**Pull request by hardbyte**
Does the manual component of #99.

I just selected all files and asked pycharm to clean em up

### CODE REVIEW

File: clkhash/bloomfilter.py [hunk 1]
```diff
@@ -30,11 +30,11 @@
...
-def double_hash_encode_ngrams(ngrams,          # type: Iterable[str]
-                              keys,            # type: Sequence[bytes]
-                              k,               # type: int
-                              l,               # type: int
-                              encoding         # type: str
+def double_hash_encode_ngrams(ngrams,  # type: Iterable[str]
+                              keys,  # type: Sequence[bytes]
+                              k,  # type: int
+                              l,  # type: int
+                              encoding  # type: str
```
Comment (nbgl) [inline clkhash/bloomfilter.py]: In Mypy’s [Python 2 examples](http://mypy.readthedocs.io/en/latest/python2.html), the type annotations are aligned like in the original.

File: clkhash/bloomfilter.py [hunk 2]
```diff
@@ -5,21 +5,21 @@
...
 import base64
-from enum import Enum
-from functools import partial
-from hashlib import md5, sha1
 import hmac
 import math
 import struct
+from enum import Enum
+from functools import partial
+from hashlib import md5, sha1
```
Comment (nbgl) [inline clkhash/bloomfilter.py]: Seems that it’s listing namespace imports (`import foo`) before from imports (`from foo import bar`). This behaviour doesn’t seem to be part of PEP8. Looks better though!

File: clkhash/schema.py [hunk 3]
```diff
@@ -188,15 +188,14 @@ def from_json_file(cls, schema_file, validate=True):
         try:
             schema_dict = json.load(schema_file)
         except ValueError as e:  # In Python 3 we can be more specific
-                                 # with json.decoder.JSONDecodeError,
-                                 # but that doesn't exist in Python 2.
+            # with json.decoder.JSONDecodeError,
+            # but that doesn't exist in Python 2.
```
Comment (nbgl) [inline clkhash/schema.py]: Yeah, fair enough. This comment should be on its own line in the first place.

File: clkhash/schema.py [hunk 4]
```diff
@@ -217,14 +216,14 @@ def get_master_schema(version):
 
     try:
         schema_bytes = pkgutil.get_data('clkhash',
-            'master-schemas/{}'.format(file_name))
+                                        'master-schemas/{}'.format(file_name))
```
Comment (nbgl) [inline clkhash/schema.py]: Good!

File: clkhash/bloomfilter.py (hunk 1 above)
Comment (hardbyte) [reply clkhash/bloomfilter.py]: I had a quick look and it didn't appear to be configurable 👎 

I'm not particularly keen to go through and manually edit... if you feel strongly about it do you want to make any changes (or look into other tools that might reformat more optimally)

**Comment by hardbyte** [comment]
As this got left behind and we had another PR which addressed code formating I've force pushed. Would you mind taking another quick look @nbgl

**Pull request merged**